airly/__init__.py

```diff
@@ -48,3 +48,13 @@ def create_measurements_session_point(
         return MeasurementsSession(
             self._rh, MeasurementsSession.Mode.POINT,
             latitude=latitude, longitude=longitude)
+
+    @property
+    def requests_remaining(self):
+        """Returns the remaining number of requests."""
```

Comment:
Would be great if you could document default values (None) for both properties if rate limits were not fetched from the server. Your finding about responses with status 200 and missing rate limits info is interesting and worth documenting as well.

Reply:
Done. I noticed that the returned values are strings so I added an integer conversion.